### PR TITLE
New upi localisation

### PR DESCRIPTION
**Pull request by neelSharma12**

### CODE REVIEW

File: AdyenComponents/UPI/UPIComponent.swift [hunk 1]
```diff
@@ -67,6 +67,10 @@ public final class UPIComponent: PaymentComponent,
 
     internal var currentSelectedIndex: Int = 0
 
+    private var hasApps: Bool {
```
Comment (descorp) [inline AdyenComponents/UPI/UPIComponent.swift:70]: Nice :)

**Review by descorp — APPROVED** [review]

**Review by erenbesel — APPROVED** [review]

File: Adyen/Assets/Generated/LocalizationKey.swift [hunk 2]
```diff
@@ -1,7 +1,8 @@
 //
-// Copyright (c) 2024 Adyen N.V.
+// Copyright (c) 2023 Adyen N.V.
```
Comment (atmamont) [inline Adyen/Assets/Generated/LocalizationKey.swift]: Do we have year hardcoded somewhere?

**Review by atmamont — APPROVED** [review]

File: Adyen/Assets/Generated/LocalizationKey.swift (hunk 2 above)
Comment (neelSharma12) [reply Adyen/Assets/Generated/LocalizationKey.swift]: This is interesting. Let me check on this.

Comment (neelSharma12) [reply Adyen/Assets/Generated/LocalizationKey.swift]: In my local copy now it is showing changes again but the 2024 one 🤔 . I did nothing

**Pull request merged**